sources/Launcher/DYNMarginCalculationLauncher.h

```diff
@@ -310,7 +309,7 @@ class MarginCalculationLauncher : public RobustnessAnalysisLauncher {
   std::map<double, ScenarioStatus, dynawoDoubleLess> scenarioStatus_;  ///< Map of scenario status by variation
 
 
-  std::vector<LoadIncreaseResult> results_;  ///< results of the systematic analysis
+  std::map<double, LoadIncreaseResult> results_;  ///< results of the systematic analysis
```

Comment:
I don't see why we need a map here

Reply:
Since the order of the simulations (load increase + scenarios) wasn't important I thought it could be more logical to use a map than a vector but I may be wrong.

Reply:
The order is important, please keep a vector